src/wasm-builder.h

```diff
@@ -981,7 +984,7 @@ class Builder {
   }
 
   static void clearLocals(Function* func) {
-    func->sig.params = Type::none;
+    func->type = Signature(Type::none, func->getResults());
     func->vars.clear();
     clearLocalNames(func);
   }
```

Comment:
huh, this is an odd API... removing params as it does.

It looks like there are no uses of it anywhere. I suggest we delete it.